ui/apps/dashboard/src/components/Navigation/Alert.tsx

```diff
@@ -34,7 +34,7 @@ export const Alert = () => {
             onClick={() => dismiss()}
           />
         </div>
-        <Link href="http://www.inngest.com/blog/reimaging-our-ia" className="mt-4">
+        <Link href="http://inngest.com/blog/inngest-new-design" className="mt-4">
```

Comment:
```suggestion
        <Link href="https://www.inngest.com/blog/inngest-new-design" className="mt-4">
```

The site is on www subdomain now and just added https :)

Reply:
thanks!